JWTDecode.podspec

```diff
@@ -16,7 +16,8 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '11.0'
```

Comment:
We should update the deployment targets in the Podspec as well.

Reply:
Missed it, updated now